#!/bin/sh /usr/share/dpatch/dpatch-run
## 325-magic-add-ssl.dpatch by Nicolas Collignon <tsointsoin@gmail.com>
##
## DP: New magic for OpenSSL certificates/key files (Closes: #439537).

@DPATCH@

diff -Naur file-4.21.orig/magic/Magdir/ssl file-4.21/magic/Magdir/ssl
--- file-4.21.orig/magic/Magdir/ssl	1970-01-01 00:00:00.000000000 +0000
+++ file-4.21/magic/Magdir/ssl	2007-08-26 08:27:04.000000000 +0000
@@ -0,0 +1,7 @@
+# Type: OpenSSL certificates/key files
+# From: Nicolas Collignon <tsointsoin@gmail.com>
+
+0	string	-----BEGIN\ CERTIFICATE-----	PEM certificate
+0	string	-----BEGIN\ CERTIFICATE\ REQ	PEM certificate request
+0	string	-----BEGIN\ RSA\ PRIVATE	PEM RSA private key
+0	string	-----BEGIN\ DSA\ PRIVATE	PEM DSA private key
